packages/start/src/server-handler/api/index.ts

```diff
@@ -0,0 +1,25 @@
+/// <reference types="vinxi/types/server" />
+import { eventHandler, toWebRequest } from 'vinxi/http'
+import { getManifest } from 'vinxi/manifest'
+import type { HTTPEvent } from 'vinxi/http'
+
+async function handleServerRequest(event: HTTPEvent) {
+  const request = toWebRequest(event)
+
+  const url = new URL(request.url, 'http://localhost:3000')
+
+  let filepath = url.pathname
+  if (filepath.startsWith('/api')) {
+    filepath = filepath.slice(4)
+  }
+
+  const method = request.method.toUpperCase()
+
+  const requestHandler = (
+    await getManifest('api').chunks[filepath]?.import()
```

Comment:
This is what's currently breaking.

In `dev`:
- It's trying to get the API route handler from the root.
```
package.json
[...] trying to search here
```

In `prod`:
- It's trying to import `.mjs` files, whilst the output is `.js`.
```
.vinxi/
  build/
    api/
      api/
        api.js
        hello-world.js
        .vite/
        // ...
```

This is what's in the manifest:
```
{
  "app/routes/api/hello-world.ts?pick=GET": {
    "file": "hello-world.js",
    "name": "hello-world",
    "src": "app/routes/api/hello-world.ts?pick=GET",
    "isEntry": true
  },
  "app/routes/api/hello-world.ts?pick=POST": {
    "file": "hello-world2.js",
    "name": "hello-world",
    "src": "app/routes/api/hello-world.ts?pick=POST",
    "isEntry": true
  },
  "virtual:$vinxi/handler/api": {
    "file": "api.js",
    "name": "api",
    "src": "virtual:$vinxi/handler/api",
    "isEntry": true
  }
}
```